priv/perf/config/dev.exs

```diff
@@ -1,6 +1,28 @@
 use Mix.Config
 
+config :ethereumex,
+  url: System.get_env("ETHEREUM_RPC_URL")
+
 config :load_test,
-  child_chain_url: "https://dev-7c3f796-childchain-ropsten-01.omg.network/",
-  watcher_security_url: "https://dev-7c3f796-watcher-ropsten-01.omg.network/",
-  watcher_info_url: "https://dev-7c3f796-watcher-info-ropsten-01.omg.network/"
+  child_chain_url: System.get_env("CHILD_CHAIN_URL") || "https://stress-e043a92-childchain-ropsten-01.omg.network",
+  fee_wei: 1,
+  faucet_deposit_wei: trunc(:math.pow(10, 18)),
+  faucet_account: %{
+    addr: "0x9133f35d9a964c894f152c0e7da66e832735a7a6",
+    priv: "0x70ad9d48f90430607a340ea2e00495e1f84d2c50a4a3df0917e6161045b32378"
+  },
+  # 0.0000001 ETH
+  initial_funds_wei: trunc(:math.pow(10, 10))
+
+config :ex_plasma,
```

Comment:
this is an obvious mistake @achiurizo , I don't think this should live in ex_plasma.